### PR TITLE
Support sdk_transaction

### PR DESCRIPTION
**Pull request by hewigovens**

### CODE REVIEW

**Review by leoneparise — APPROVED** [review]

**Review by ArtemGoryunov — APPROVED** [review]

File: TrustSDK/Classes/Client/Types/UniversalAssetID.swift [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import Foundation
+import WalletCore
+
+/// https://developer.trustwallet.com/add_new_asset/universal_asset_id
+public struct UniversalAssetID: CustomStringConvertible, Equatable {
+
+    let cPrefix = "c"
```
Comment (vikmeup) [inline TrustSDK/Classes/Client/Types/UniversalAssetID.swift:13]: `cPrefix` => `coinPrefix`
`tPrefix` => `tPrefix`

File: Example/Tests/CommandTests.swift [hunk 2]
```diff
@@ -105,4 +105,54 @@ class CommandTests: XCTestCase {
...
+        let wallet = WalletApp(scheme: app, installURL: URL(string: "https://trustwallet.com")!)
+        let tx = TrustSDK.Transaction(
+            asset: UniversalAssetID(coin: .ethereum),
+            to: "0x728B02377230b5df73Aa4E3192E89b6090DD7312",
+            amount: "0.01",
+            action: .transfer,
+            confirm: .sign,
+            from: nil,
+            nonce: 447,
+            feePrice: "2112000000",
```
Comment (vikmeup) [inline Example/Tests/CommandTests.swift:120]: Can we use `BigInt` here?

File: TrustSDK/Classes/Client/Types/UniversalAssetID.swift [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+//
+// This file is part of TrustSDK. The full TrustSDK copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import Foundation
+import WalletCore
+
+/// https://developer.trustwallet.com/add_new_asset/universal_asset_id
+public struct UniversalAssetID: CustomStringConvertible, Equatable {
```
Comment (vikmeup) [inline TrustSDK/Classes/Client/Types/UniversalAssetID.swift:11]: Add tests for `UniversalAssetID`

File: TrustSDK/Classes/Client/Types/Transaction.swift [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        }
+
+        public let to: String
+        public let asset: UniversalAssetID
+        public let amount: String
+        public let action: Action
+        public let confirm: ConfirmType
+        public let from: String?
+        public let nonce: UInt64?
+        public let feeLimit: String?
```
Comment (vikmeup) [inline TrustSDK/Classes/Client/Types/Transaction.swift:22]: should we use `BigInt` for amount, fee and limit?

Comment (vikmeup) [reply TrustSDK/Classes/Client/Types/Transaction.swift:22]: and maybe nonce even

Comment (hewigovens) [reply TrustSDK/Classes/Client/Types/Transaction.swift:22]: yes, `BigInt` for `amount`, `fee`, `limit`, `UInt64` is enough for `nonce`

File: TrustSDK/Classes/Client/Types/UniversalAssetID.swift (hunk 3 above)
Comment (hewigovens) [reply TrustSDK/Classes/Client/Types/UniversalAssetID.swift:11]: https://github.com/trustwallet/TrustSDK-iOS/blob/master/Example/Tests/AssetIDTests.swift

**Pull request merged**